core/server/apps/subscribers/index.js

```diff
@@ -20,25 +20,35 @@ var _          = require('lodash'),
      *   document.querySelector('.referrer').setAttribute('value', document.referrer);
```

Comment:
Hey, can you plz update the comment here with the non-minified version of what the script does? 👍